examples/SampleApp/src/hooks/usePaginatedUsers.ts

```diff
@@ -153,33 +150,35 @@ export const usePaginatedUsers = (): PaginatedUsers => {
         },
       );
 
-      if (!res?.users) {
-        queryInProgress.current = false;
-        return;
-      }
+      const usersWithoutClientUserId = users.filter((user) => user.id !== chatClient.userID);
 
       // Dumb check to avoid duplicates
-      if (query === searchText && results.findIndex((r) => res?.users[0].id === r.id) > -1) {
+      if (
+        query === searchText &&
+        usersWithoutClientUserId.length > 0 &&
+        results.findIndex((r) => usersWithoutClientUserId[0].id === r.id) > -1
+      ) {
         queryInProgress.current = false;
         return;
       }
```

Comment:
Again unrelated but, this is quite confusing to me. What does this code do?

Reply:
We can remove it imo. Seems useless now